### DISCUSSION TITLE
model input file is not readable

### DISCUSSION THREAD

**Discussion by tangshuaiqi** [ARM-Development/comble-mip#29] [Q&A] [answered]
The model input file, `COMBLE_INTERCOMPARISON_FORCING_V2.3.nc`, is not able to be seen via ncdump or xarray in python:

`ncdump: COMBLE_INTERCOMPARISON_FORCING_V2.3.nc: NetCDF: Unknown file format`

**Comment by fridlind** [comment]
Did you download the file directly from the COMBLE-MIP repo? The attached screenshot shows how I just did that. When I chose "download" and check it on my desktop, it is readable as expected. Please let us know if you experience any further difficulty.
<img width="1416" alt="Screenshot 2023-12-06 at 11 47 09 AM" src="https://github.com/ARM-Development/comble-mip/assets/20172705/c91e5fd9-8311-49ae-987a-bc085ddbca3d">

**Comment by tangshuaiqi** [comment]
yes I downloaded directly from the repo. I just copied it to NERSC and PNNL supercomputer and try to use ncdump to see the filehead, but get the error in my question. I also tried to open it locally using xarray (but I am using Spyder) and get the error below. I already have netCDF4 imported...
![image](https://github.com/ARM-Development/comble-mip/assets/40275450/decceff1-42be-42a9-83d6-5fc47169de43)

**Comment by fridlind** [comment]
We briefly had a corrupted file there, but that was corrected about a week ago. Assuming that you downloaded the file after the intercomparison was opened on Friday, there should have been no problem. I have placed another copy of the file temporarily on a share drive for you [here](https://drive.google.com/file/d/1qjEKByai0Ymxu01FaJkWBwOGPyEr-qJe/view?usp=sharing). Can you please grab this file and check it on your systems? If it works, there is likely a download side problem. If not, there is likely an environment problem. Thanks!

**Comment by tangshuaiqi** [comment]
This one works. And the size is 172KB while the bad file I downloaded directly from the repo (even today) is 407KB.

**Comment by fridlind** [comment]
To my knowledge, there was no relevant change to the forcing file contents since last Friday. If you click on the filename in your screenshot above, you will obtain my screenshot shown above, which lists its size as 171 KB. I think something happened to corrupt the file during your download, but I'm not sure what. Thank you for alerting us to this.

**Comment by tangshuaiqi** [comment, marked as answer]
I found an interesting issue that when I click on the filename and download as in your screenshot, the file is good with size of 171KB. But if I right click the filename and choose "save link as", the file is 407 KB and is bad.

**Comment by fridlind** [comment]
OK, that's good to know, thanks. 

In case you model does not read in the DEPHY format, by the way, it is also possible to use the ARM Workbench and our example scripts (under "Input Conversion Notebooks") as a template to create your own LES or SCM input files in the format required by your model. To use the ARM Workbench, first [apply for access](https://arm-development.github.io/comble-mip/notebooks/howto/workbench_instructions.html). Then, from any COMBLE-MIP script page, hover your cursor over the little rocket ship icon (over the word 'forcing' below) in order to launch a JupyterHub session on the ARM Workbench (as further described [here](https://arm-development.github.io/comble-mip/CONTRIBUTING.html)).

![image](https://github.com/ARM-Development/comble-mip/assets/20172705/a0c9dbe5-7854-4e05-bb2d-bedccda836e4)
